forecastio.class.nut

```diff
@@ -34,11 +34,12 @@ class Forecastio {
         _apikey = key;
     }
 
-    function forecastRequest(longitude = 999, latitude = 999, callback = null) {
+    function forecastRequest(longitude = 999, latitude = 999, units = us, callback = null) {
         // Parameters:
         //  1. Longitude of location for which a forecast is required
         //  2. Latitude of location for which a forecast is required
-        //  3. Optional synchronous operation callback
+        //  3. Optional units parameter which defaults to us is not specified
```

Comment:
Should be:
//  3. Optional units parameter which defaults to "us" if not specified